source/index.txt

```diff
@@ -19,46 +19,86 @@ MongoDB Go Driver
 Introduction
 ------------
 
-The official MongoDB Go Driver enables Go applications to connect to
-MongoDB and work with data.
+The official MongoDB Go Driver enables Go applications to connect to and
+work with data in MongoDB. On this page, you can find descriptions of
+each section of the driver documentation.
+
+Learn
+-----
+
+Visit the Developer Hub for content, resources, community, and social
+engagement for developers related to the Go driver.
+
+To view articles on Go, see the `How To's and Articles page
+<https://www.mongodb.com/developer/learn/?content=Articles&text=Go>`_.
+
+To view the forum on Go, see the `forums page
+<https://www.mongodb.com/community/forums/tag/golang>`_.
+
+Quick Start
+-----------
+
+To get up and running quickly, take a look at the :doc:`Quick Start
+</quick-start>` to learn how to establish a connection to MongoDB Atlas
```

Comment:
Nit: I would be more specific than establishing a connection and would break this into 2 sentences:

```
To get up and running quickly, take a look at the :doc:`Quick Start
</quick-start>`, which shows you how to: 

- Create an application,
- Use the Go driver to connect to MongoDB Atlas, and
- Begin to work with data.
```